lib/connectors/crawler/scheduler.rb

```diff
@@ -22,11 +22,48 @@ def connector_settings
         []
       end
 
+      def when_triggered
+        loop do
+          connector_settings.each do |cs|
+            # crawler only supports :sync
+            if sync_triggered?(cs)
+              yield cs, :sync, nil
+              next
+            end
+
+            custom_schedule_triggered, schedule_key = custom_schedule_triggered?(cs)
+            if custom_schedule_triggered
+              yield cs, :sync, schedule_key
```

Comment:
Do you also plan to propagate `schedule_key` to the job's payload?

Reply:
Do you mean to the `.elastic-connectors-sync-jobs` doc for this sync job? If so, yes.